grayskull/pypi/pypi.py

```diff
@@ -581,12 +598,14 @@ def _get_metadata(self) -> dict:
         print_msg(f"License file: {Fore.LIGHTMAGENTA_EX}{license_file}")
 
         all_requirements = self._extract_requirements(metadata)
```

Comment:
Maybe we need to look into why this function is _not_ returning a dictionary populated as expected, and we fix that misbehaviour in that function, instead of patching it here.

Reply:
(optional)